app/lib/screens/HomeScreens/chat/ChatScreen.dart

```diff
@@ -227,14 +232,25 @@ class _ChatScreenState extends State<ChatScreen> {
         child: Image.file(
           File(imageMessage.uri),
           width: messageWidth.toDouble(),
-          errorBuilder: (context, error, stackTrace) => const Text(
-            'Could not load image',
-          ),
+          errorBuilder: (
+            BuildContext context,
+            Object error,
+            StackTrace? stackTrace,
+          ) {
+            return const Text('Could not load image');
+          },
         ),
       );
     }
   }
 
+  Widget customMessageBuilder(
+    types.CustomMessage customMessage, {
+    required int messageWidth,
+  }) {
+    return const SizedBox();
```

Comment:
@bitfriend Try replacing with Placeholder text.

Reply:
Also we want to hide receipts but not bubble. Does unsupported message in matrix terms returns something? Decryption failed error?

Reply:
I don't know, but we can use the same text as element app.